### ISSUE TITLE
Element accessibility id, name, or label value is not showing on iOS 15.0

### ISSUE THREAD

**Issue by sshaonvet29** [appium/appium#16679]
## The problem
Element accessibility id, name, or label value is not showing on iOS 15.0

## Environment

* Appium version (or git revision) that exhibits the issue: 1.22.0
* Last Appium version that did not exhibit the issue (if applicable):1.22.0
* Desktop OS/version used to run Appium: Mac
* Node.js version (unless using Appium.app|exe):
* Npm or Yarn package manager:
* Mobile platform/version under test:
* Real device or emulator/simulator:simulator
* Appium CLI or Appium.app|exe:

## Details

Hi, we are automating all our iOS automated tests using iOS version 14.3 and Appium version 1.22.0. We are planning to update our iOS version from 14.3 to 15.0 or 15.2. But tests are failed because Appium is unable to find the given element property(like accessibility id, name, label, or value) which is completely fine on iOS 14.3. Appium Inspector is also not showing the id, label, name, or value for those elements for iOS 15. And also in some cases, the element tree is not loading properly. It looks like an iOS issue to me than Appium.
Just wondering, if anybody has experienced the same problem with their automated tests in iOS when updating the version of iOS from 14 to 15.
Thanks in advance!

## Link to Appium logs

Create a [GIST](https://gist.github.com) which is a paste of your _full_ Appium logs, and link them here.
Do _NOT_ paste your full Appium logs here, as it will make this issue very long and hard to read!
If you are reporting a bug, _always_ include Appium logs!

## Code To Reproduce Issue [ Good To Have ]

Please remember that with sample code it's easier to reproduce the bug and it's much faster to fix it.

Please git clone https://github.com/appium/appium and from the `sample-code` directory, use one of your favourite languages and sample apps to reproduce the issue.

In case a similar scenario is missing in sample-code, please submit a PR with one of the sample apps provided.

**Comment by mykola-mokhnach** [comment]
I also tend to think something has changed in XCTest itself. You could verify that by retrieving the page source directly from XCTest and comparing the output between different OS versions. [mobile: source](https://github.com/appium/appium-xcuitest-driver#mobile-source) extensions could be used for this purpose with `format` arg set to `description`

**Comment by mykola-mokhnach** [comment]
Closed as third party issue
